packages/slate-editor/src/modules/editor-v4-rich-formatting-menu/LinkMenu.tsx

```diff
@@ -11,13 +11,22 @@ import { STRING_URL_PATTERN } from '#modules/editor-v4-components/LinkMenu';
 interface Props {
     node: LinkNode | null;
     canUnlink: boolean;
+    showOpenInNewTabOption: boolean;
```

Comment:
I suggest unifying the property name. 

Right now the same thing has three different names in different components:

- `linksOpenInNewTabOption`
- `showOpenInNewTabOption`
- `withLinksOpenInNewTabOption`

How about renaming it to `withNewTabOption` everywhere?

Reply:
done